extensions/dontfragment/ipt_DF.c

```diff
@@ -10,6 +13,12 @@ MODULE_AUTHOR("Semyon Verchenko");
 MODULE_DESCRIPTION("Netfilter module to set/reset DF flag");
 MODULE_LICENSE("Dual BSD/GPL");
 
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(5,2,0))
+# skb_make_writable was removed in this commit:
+# https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit?id=2cf6bffc49dae26edd12af6b57c8c780590380bf
```

Comment:
Comments should begin with `/*` :-)

Reply:
Also... i see `skb_ensure_writeable` is present in 4.15. Is this needed ?

Reply:
@fpopescu-dn Using `skb_ensure_writeable()` was actually a bug, and so it appears we need to use `skb_make_writable()` https://github.com/drivenets/dontfragment/commit/38e1b4d550b55003f4b0fbe9b947c560963bc71f
\+ @oshamash might have more info

Reply:
I remember now @oshamash pinged me about this some time ago (spending days to find this). So why it would work now in the 5.2+

Reply:
Sorry, it was this https://github.com/drivenets/dontfragment/commit/94d2273023589b9c83e20b0ddce5aa65bac13be5

Reply:
> Comments should begin with `/*` :-)

Fixed

Reply:
I made a dumb compat fix assuming code is already correct. Seeing how it doesn't even compile on 5.4 it might be unused.